### PR TITLE
integrate overlays into resource actions

### PR DESCRIPTION
**Pull request by xi**
This implements overlays directly in resource actions.

Should work while keeping compatibility with legacy system.

TODO: static layout for alerts.

### CODE REVIEW

**Comment by der-john** [comment]
very nice! looks great & works.

File: src/adhocracy_frontend/adhocracy_frontend/static/js/Packages/ResourceActions/ResourceActions.html [hunk 1]
```diff
@@ -34,4 +36,24 @@
...
+        {{ alert.message | translate }}
+    </li>
+</ul>
+
+<div class="action-bar-modal" data-ng-if="modals.overlay">
+    <adh-report-abuse
+        data-ng-if="modals.overlay === 'abuse'"
+        data-modals="modals"
+        class="report-abuse"
+        data-url="{{ proposalUrl | adhResourceUrl | adhCanonicalUrl }}">
```
Comment (der-john) [inline src/adhocracy_frontend/adhocracy_frontend/static/js/Packages/ResourceActions/ResourceActions.html]: Why did you leave `proposalUrl` and didn't change it to `resourcePath`? (It seems to work fine, but I thought `resourceActions` are not just about proposals.)

Comment (xi) [inline src/adhocracy_frontend/adhocracy_frontend/static/js/Packages/ResourceActions/ResourceActions.html]: Indeed this is an error. I will fix it.

**Comment by der-john** [comment]
Is it possible that we forgot `adh-share-action` here as well?

**Comment by xi** [comment]
> Is it possible that we forgot `adh-share-action` in bcd399e?

As far as I can see it is not used anywhere outside of AdhResourceActions

**Comment by der-john** [comment]
> Is it possible that we forgot adh-share-action in bcd399e?
> > As far as I can see it is not used anywhere outside of AdhResourceActions

You're right.

**Pull request merged**